topics/metabolomics/tutorials/msi-finding-nglycans/tutorial.md

```diff
@@ -157,27 +157,27 @@ To get an overview of the combined dataset we will use the **MSI Qualitycontrol*
+The quality report gives a broad overview of the properties of both files. The pixel order seems to be random and does not correspond to the measurement in the MALDI. This means that the plots where the 'spectra index' is plotted on the x-axis do represent the acquisition process. While the PNGase F treated kidney file shows higher intensities in several plots (e.g. median intensity per spectrum and intensity boxplots), the number of peaks is similar in both subfiles. The most abundant m/z feature plot of the treated kidney roughly corresponds to the morphology of the kidney while in the control file there are no visible kidney structures. The first principal componant in the PCA analysis roughly differentiates the spectra of the two subfiles which is a good sign for finding differences in the treated vs. control kidney files. For the following preprocessing steps, it is important to know the rough number of data points that one peak consists of. From page 27 on, the average mass spectra of the two glycan calibrants are shown. Due to our harsh resampling to 0.1 m/z, the calibrant 1257.4225 (Man5GlcNAc2) shows quite an angular peak shape. The complete peak consists of about 6 data points, which corresponds to a width of 0.5 m/z. On page 28 the average spectrum for each subfile is plotted, showing a nice peak in the treated subfile and only basal intensity in the control subfile. Page 29 and 30 show the average mass spectra for the 1542.5551 (Man3GlcNAc5) calibrant. The intensities of this calibrant are relatively low and only slightly higher in the treated than in the control subfile. Furthermore, the split up peak shape is not useful for defining the peak width properly. 
```

Comment:
I didn't find this paragraph so clear. In the third sentence, do you want to say 'do represent' or 'do not'?
Also, what do the page numbers refer to? I guess the PDF that is produced by the MSI quality control tool, but this seems to have less than 27 pages.

Reply:
Could one of you maybe comment on this @lecorguille @foellmelanie ? The page numbers in particular are strange. When I tested the tool I got a PDF with only 17 pages.

Reply:
@simonbray Sorry I was called away during reviewing and did not manage to continue. You are right, it should be 'do not represent'. You probably you tested the training with the workflow? There is a connection missing between the calibrants and the quality control. With the calibrant inputs the PDF has 35 pages. Sorry,  I will update the workflow very soon. 
Thanks a lot for testing and reviewing! :+1:

Reply:
You're welcome! Yes, you're right, I missed that step.